packages/next-on-pages/templates/_worker.js/utils/routing.ts

```diff
@@ -131,7 +131,7 @@ export async function runOrFetchBuildOutputItem(
 		return new Response('Internal Server Error', { status: 500 });
 	}
 
-	return createMutableResponse(resp);
+	return resp;
```

Comment:
`generateResponse` uses this function, but it already does what `createMutableResponse` does when needed.

It is also used here `packages/next-on-pages/templates/_worker.js/routes-matcher.ts:239`, but since my app is not using middlewares I couldn't test it, `npm run test:unit` passes though.
If it is needed there, maybe we can do `createMutableResponse` locally for it?

Hopefully my changes are not fundamentally against your design, I just went with common sense based on my very limited knowledge on the codebase.

Reply:
I changed it to
```typescript
	if (resp.status === 101) {
		return resp;
	}
	return createMutableResponse(resp);
```

again this is an attempt to allow websocket connections without knowing much about the codebase, I don't know if it is only status code 101 we should put this exception for, or if it is the correct signal for the exception.